helpdesk_mgmt/views/report_helpdesk_ticket.xml

```diff
@@ -0,0 +1,104 @@
+                        <t>
+                            <span>Ticket #</span>
+                        </t>
+                        <span t-field="doc.number" />
+                    </h2>
+
+                    <h3 class="mt16">
+                        <span t-field="doc.name" />
+                    </h3>
+
+                    <div class="row mt32 mb32" id="informations">
+                        <div t-if="doc.team_id" class="col-auto mw-100 mb-2">
+                            <strong>Team:</strong>
+                            <p class="m-0" t-field="doc.team_id" />
+                        </div>
+
+                        <div t-if="doc.user_id" class="col-auto mw-100 mb-2">
+                            <strong>Assigned user:</strong>
+                            <p class="m-0" t-field="doc.user_id" />
+                        </div>
+
+                        <div t-if="doc.create_date" class="col-auto mw-100 mb-2">
+                            <strong>Create on:</strong>
+                            <p class="m-0" t-field="doc.create_date" />
+                        </div>
+
+                        <div t-if="doc.closed_date" class="col-auto mw-100 mb-2">
+                            <strong>Closed Date:</strong>
+                            <p class="m-0" t-field="doc.closed_date" />
+                        </div>
+
+                        <div t-if="doc.priority" class="col-auto mw-100 mb-2">
+                            <strong>Priority:</strong>
+                            <p class="m-0" t-field="doc.priority" />
+                        </div>
+
+                        <div t-if="doc.stage_id" class="col-auto mw-100 mb-2">
+                            <strong>Stage:</strong>
+                            <p class="m-0" t-field="doc.stage_id" />
+                        </div>
+                    </div>
+
+                    <div class="row mt32 mb32" id="ticket_informations">
+                        <div t-if="doc.channel_id" class="col-auto mw-100 mb-2">
+                            <strong>Channel:</strong>
+                            <p class="m-0" t-field="doc.channel_id" />
+                        </div>
+
+                        <div t-if="doc.category_id" class="col-auto mw-100 mb-2">
+                            <strong>Category:</strong>
+                            <p class="m-0" t-field="doc.category_id" />
+                        </div>
+
+                        <div t-if="doc.tag_ids" class="col-auto mw-100 mb-2">
+                            <strong>Tag:</strong>
+                            <p class="m-0" t-field="doc.tag_ids" />
+                        </div>
+                    </div>
+
+                    <div class="row mt32 mb32" id="description">
+                        <div t-if="doc.description" class="col-auto mw-100 mb-2">
+                            <h3>Description:</h3>
+                            <p class="m-0" t-field="doc.description" />
+                        </div>
```

Comment:
I think there's something wrong here with the description width. If you print a report with a long description, you can see that only two columns are filled

Reply:
I didn't notice the problem. Take a look at the file.
![image](https://user-images.githubusercontent.com/3595132/87327259-39e6e700-c50a-11ea-89d9-17944a9a07ec.png)

Reply:
Well, I've tested on runbot, this is the result with a demo ticket data (rendered by Firefox):

![imagen](https://user-images.githubusercontent.com/12749832/87328609-06ba4d00-c536-11ea-97e1-96a2e3d95195.png)

The related demo data, unchanged:

![imagen](https://user-images.githubusercontent.com/12749832/87328691-26ea0c00-c536-11ea-8c0b-bf2e99bee44f.png)

It should be expected to fill only one row within the report, shouldn't it be?

Reply:
Now it looks right

Reply:
@dalonsod Try again please.